src/sentry/tasks/post_process.py

```diff
@@ -172,7 +172,7 @@ def index_event_tags(project_id, event_id, tags, group_id=None, **kwargs):
         )
 
         tagvalue, _ = TagValue.objects.get_or_create(
-            project=Project(id=project_id),
+            project=Project.objects.get(id=project_id),
```

Comment:
sooo it originally was blowing up here: https://github.com/getsentry/sentry/pull/4627/files#diff-0b02ede14b5c5c0c44866262897ee8a0R21 when i ran `bin/load-mocks` but now can't actually reproduce that, so must have had something else wrong that i accidentally fixed :-/ will revert this back to what it was